source/administration/config-settings.rst

```diff
@@ -473,19 +492,27 @@ Restrict the permission level required to delete messages. Team Admins, Channel
 
 Allow users to edit their messages
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Set the time limit that users have to edit their messages after posting.
+*This permission has been migrated to the database and changing the config.json value no longer takes effect after upgrading to v4.9, released on April 16th, 2018. This permission can be modified using the System Console user interface.*
+
+Set whether users can edit their messages after posting.
 
 **Any time**: Allow users to edit their messages at any time after posting.
 
 **Never**: Do not allow users to edit their messages.
 
 **{n} seconds after posting**: Users can edit their messages within the specified time limit after posting.
 
-+---------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| This feature has two settings in ``config.json``. The first setting is ``"AllowEditPost": "always"`` with options ``always``, ``never``, and ``time_limit``.  |
-|                                                                                                                                                               |
-| The second setting is ``"PostEditTimeLimit": -1`` with whole number input. To enable ``PostEditTimeLimit``, set ``AllowEditPost`` to ``time_limit``.          |
-+---------------------------------------------------------------------------------------------------------------------------------------------------------------+
++------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"AllowEditPost": "always"`` with options ``always``, ``never``, and ``time_limit`` for above settings respectively.  |
++------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Post edit time limit
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When post editing is enabled, set the time limit users have to edit their messages.
+
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"PostEditTimeLimit": -1`` with whole number input. When post editing is permitted, set ``"PostEditTimeLimit": -1`` to allow editing anytime, or set ``"PostEditTimeLimit"`` to a positive integer to restrict editing time in seconds. If post editing is disabled, this setting does not apply.  |
```

Comment:
Should the longer description below be part of the setting description rather than inside the textbox?

```
When post editing is permitted, set ``"PostEditTimeLimit": -1`` to allow editing anytime, or set ``"PostEditTimeLimit"`` to a positive integer to restrict editing time in seconds. If post editing is disabled, this setting does not apply.
```